src/StatusBar/index.js

```diff
@@ -11,14 +11,16 @@ export function StatusBar({
   caretPosition = -1,
   sequenceLength = 0,
   readOnly,
-  sequenceData: { circular } = {},
+  sequenceData: { circular, materiallyAvailable } = {},
```

Comment:
@juFelipe should this really be a property on the sequenceData? It seems like it should live outside of sequenceData to be more akin to say, `readOnly`.